packages/components/src/toolbar-group/test/index.js

```diff
@@ -76,15 +76,17 @@ describe( 'ToolbarGroup', () => {
 				],
 			];
 
-			const { container } = render(
-				<ToolbarGroup controls={ controls } />
-			);
+			render( <ToolbarGroup controls={ controls } /> );
 
 			const buttons = screen.getAllByRole( 'button' );
+
 			expect( buttons ).toHaveLength( 2 );
-			expect(
-				container.querySelector( '.has-left-divider button' )
-			).toBe( buttons[ 1 ] );
+			expect( buttons[ 0 ].parentElement ).not.toHaveClass(
```

Comment:
Fixing it that way will create a `no-node-access` violation, but we'll take care of that separately by either ignoring it or fixing it. One step at a time!